### ISSUE TITLE
TPAC planning and specification status report

### ISSUE THREAD

**Issue by LJWatson** [w3c/clipboard-apis#124]
TPAC will be virtual this year. W3C is finalising the details but it is likely there will be events for the whole W3C community during the week of 26 to 31 October (as originally planned), with WG able to organise their own meetings in/around that week.

There are two things we need you to do:
1. Let @marcoscaceres  and I know before 17 July if you want meeting time to talk about your specification.
2. Post a specification status report before 30 September.

The specification status report should include:
* What progress has your spec made in the last 12 months?
* Is anything blocking your spec from moving to CR?
* If yes, what is your plan to unblock it and do you need any help?
* What do you want to achieve for your spec in the next two days (of breakout sessions)?

Your specification status report from 2019 is at #94

**Comment by dway123** [comment]
Yes, it would be great to have time to talk about the Clipboard Specification.

I'd like to discuss progress regarding the Raw Clipboard proposal last year, as well as a new, related proposal for pickling.

(Also noted in [public-editing-tf archive](https://lists.w3.org/Archives/Public/public-editing-tf/2020Oct/0012.html))

I might not have full context for a specification status report, but this specification involved discussions regarding permissions and raw clipboard over the last 12 months.

**Comment by marcoscaceres** [comment]
# What progress has your spec made in the last 12 months?

Looking at commit history, no significant normative changes have occurred in the last 12 months.

**Comment by dway123** [comment]
I've set up a time next Monday, October 26, 10PM UTC (3PM PST), to discuss Pickling for Clipboard API, and updated the WebApps WG Calendar appropriately. Please see this [public-editing-tf](https://lists.w3.org/Archives/Public/public-editing-tf/2020Oct/0014.html) archive or the [calendar link here](https://calendar.google.com/event?action=TEMPLATE&tmeid=NHFvbmgxZjQwY2RyNHZtdmV0aHQwOWVuNGsgY19tczEyajVndXFhczdnZmg5ZjRiN3YydGY1MEBn&tmsrc=c_ms12j5guqas7gfh9f4b7v2tf50%40group.calendar.google.com) for more information.

**Comment by annevk** [comment]
I'm sorry I missed that session, is there a summary available somewhere? I'm also interested in hearing about maintenance for this specification as it has quite a few old issues, including for the new APIs.

**Comment by dway123** [comment]
There are minutes available as part of the WebApps WG minutes. https://www.w3.org/2020/10/19-webapps-minutes.html#item03

**Comment by annevk** [comment]
@LJWatson @marcoscaceres it's pretty clear at this point that this specification has a maintenance problem, e.g., including IDL without a normative description of what the IDL does. Unfortunately I don't have a proposal for how to resolve that, but I thought I'd at least bring it to your attention.

**Comment by marcoscaceres** [comment]
Thanks for raising the issue @annevk. We might need to find an additional editor to help support the current editor.
